docs/templates/cli_rst.j2

```diff
@@ -128,12 +128,9 @@ See the `AUTHORS` file for a complete list of contributors.
 Copyright
 =========
 
-Copyright © 2017 Red Hat, Inc | Ansible.
-
 Ansible is released under the terms of the GPLv3 License.
```

Comment:
Maybe we should change the Header to License as well.  Also, I believe that it is GPLv3+, not GPLv3.

Reply:
oh, good catch, we didn't even notice that